easyconfigs/d/dcm2bids/dcm2bids-3.1.1-foss-2022a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'dcm2bids'
+version = '3.1.1'
+
+homepage = 'https://unfmontreal.github.io/Dcm2Bids'
+description = """dcm2bids reorganises NIfTI files using dcm2niix into the Brain Imaging Data Structure (BIDS)."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
```

Comment:
I know it's not listed in their deps but should _dcm2niix_ be supplied? For example, there's a check for it in: 
https://github.com/UNFmontreal/Dcm2Bids/blob/0819eeb16a981478b9342bc6dac826dfe5fb7081/dcm2bids/utils/tools.py#L94